plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShapeSelection.cpp

```diff
@@ -13,6 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+#include <cinttypes>
+
```

Comment:
Can this be wrapped around some #ifdef? Not sure if this applies to other platforms.

Reply:
Should be fine, it's a C++ 11 header.

Reply:
sounds good!

Reply:
OK, I understand that fixes done to remove compilation errors on OSX when building in strict mode created this issue - https://github.com/Autodesk/maya-usd/commit/c55c976fcefc07bc44f98c4e808e4089fcb60372#diff-94b4ec7e80f873f5e91609163e535682
Correct?

Reply:
Yes, and depending on which of inttypes.h or cinttypes is included first, you will get different results --- see
https://stackoverflow.com/questions/14535556/why-doesnt-priu64-work-in-this-code/30851225